/**:
  ros__parameters:
    input_image_topic: "/fv/d405/color/image_raw"
    qos:
      reliability: "best_effort"
      queue_size: 10
    backend: "openvino"
    model_path: "src/fluent_vision_ros2/models/openvino/yolov8s-seg-aspara.xml"
    device: "CPU"
    conf_thres: 0.25
    iou_thres: 0.5
    publish_detections: true
    publish_overlay: true
    nms_class_agnostic: true
    max_detections: 100
    debug_shapes: false

